### PR TITLE
Resolve fallback with all test cases

### PR DESCRIPTION
**Pull request by asvny**
Fix https://github.com/MadLittleMods/postcss-css-variables/issues/37

---

[_previous PR_](https://github.com/MadLittleMods/postcss-css-variables/pull/38)

### CODE REVIEW

**Comment by marklu** [comment]
+1

It'd be really nice if this fix lands. I've played with it a bit and it seems to work well.

File: lib/resolve-value.js [hunk 1]
```diff
@@ -5,6 +5,29 @@ var gatherVariableDependencies = require('./gather-variable-dependencies');
...
+    }
+    function copy(target, source) {
+      for (var key in source) {
+        if (has.call(source, key)) {
+          target[key] = source[key];
+        }
+      }
+    }
+  }({}.hasOwnProperty);
+}
```
Comment (MadLittleMods) [inline lib/resolve-value.js]: What are you trying to support with this?

This polyfill doesn't seem necessary for Node v4+, https://kangax.github.io/compat-table/es6/#test-Object_static_methods

Better to pull in a separate package from npm if we actually do want this.

**Comment by MadLittleMods** [comment]
@marklu Feel free to resolve the conflicts and comments in a new PR 😀

**Comment by asvny** [comment]
Ok .. I will work on the changes and make a new PR !

**Pull request closed without merging**